.infrastructure/todoapp-pod.yml

```diff
@@ -3,21 +3,50 @@ kind: Pod
 metadata:
   name: todoapp
   namespace: todoapp
+  labels:
+    app: todolist
 spec:
   containers:
   - name: todoapp
-    image: ikulyk404/todoapp:3.0.0
+    image: nemereno/todoapp:3.0.0
     ports:
     - containerPort: 8080
     livenessProbe:
       httpGet:
-        path: api/health
+        path: api/liveness
         port: 8080
       initialDelaySeconds: 60
       periodSeconds: 5
     readinessProbe:
       httpGet:
-        path: api/ready
+        path: api/readiness
         port: 8080
       initialDelaySeconds: 5
       periodSeconds: 5
+
+
+apiVersion: v1
+kind: Pod
+metadata:
+  name: todoapp-1
+  namespace: todoapp
+  labels:
+    app: todolist
+spec:
+  containers:
+  - name: todoapp
+    image: nemereno/todoapp:3.0.0
+    ports:
+    - containerPort: 8080
+    livenessProbe:
+      httpGet:
+        path: api/liveness
+        port: 8080
+      initialDelaySeconds: 60
+      periodSeconds: 5
+    readinessProbe:
+      httpGet:
+        path: api/readiness
+        port: 8080
+      initialDelaySeconds: 5
+      periodSeconds: 5
```

Comment:
Add Eof(End-of-file), need to add a blank line at the end
![image](https://github.com/user-attachments/assets/a851dbbe-6a38-4c6b-9cef-662ce216dfcf)